apps/helptheducks.com/src/app/petition.tsx

```diff
@@ -285,7 +285,7 @@ export const Petition = () => {
               <FormItem className={cn({ hidden: !cities.length })}>
                 <FormLabel>City</FormLabel>
                 <Select
-                  onValueChange={(val) => {
+                  onValueChange={(val: string | undefined) => {
```

Comment:
why do we have to do this? what is the signature of the function that `onValueChange`  wants?